src/Microsoft.ServiceFabric.Services.Queryable/Controller/ValueViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ServiceFabric.Services.Queryable.Controller
+{
+    public class ValueViewModel
```

Comment:
Can we just use Entity<JToken, JToken> instead of this class?  Delete this class.